src/types.ts

```diff
@@ -20,7 +20,7 @@ export type AirtableRecord = {
     age: string[] | null,
     denominator: number | null,
     seroprevalence: number | null,
-    publish_date?: string | null,
+    publish_date?: string[] | string | null,
```

Comment:
Are we sure it can either be a string array or a string? Double check the Airtable SOP: https://docs.google.com/spreadsheets/d/1UT0tZSj7k4OksihoIk7F3zkkoiScHzj2shkPMBF5uCQ/edit#gid=0.

Note: sometimes Airtable does this weird thing where string fields that are linked to another table show up on that other table as a single element array. If this is the case we consistently see, I would probably just map it to a string here always.

Reply:
Okay, I just had some issues

Reply:
this one I left